pkg/leader/leader.go

```diff
@@ -47,7 +47,7 @@ func Become(ctx context.Context, lockName string) error {
 
 	ns, err := k8sutil.GetOperatorNamespace()
 	if err != nil {
-		if err == k8sutil.ErrNoNamespace {
+		if err == k8sutil.ErrNoNamespace || err == k8sutil.ErrRunLocal {
```

Comment:
One question: At this point is there ever a case where `err == k8sutil.ErrNoNamespace` and `err != k8sutil.ErrRunLocal`?

Looking at the errors returned by cluster specific functions such as `initOperatorService()` and `GetOperatorNamespace()`, `GetOperatorName()` it seems they will all return `ErrRunLocal`  if running locally.

Reply:
If a user is using our commands, `ErrNoNamespace` probably shouldn't be hit since we manually force the run mode. I do assume it's possible if a user manually runs it themselves with `go run cmd/manager/main.go`.